index.html

```diff
@@ -1,14 +1,164 @@
 <html>
   <head>
-    <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <title>Remock Project</title>
+    <style>
+      @import "index.css";
+    </style>
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+  <body class="body1">
+
+    <section id="header">
+        <h1>HOME PAGE</h1> 
+        <H6>THE PAGE FOR THE HOME</H6>
+    </section>
+
+      <div class="content">
+        <h2> TAKE A LOOK AT THIS AMAZING <strong>HOME PAGE!</strong></h2>
+      </div>
+      <div id="homeimgs">
+        <div class="img1wrapper">
+          <img class="img1" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/Tree_of_life.jpg"/>
+          <div>IMAGE ONE</div>
+        </div>
+        <div class="img2wrapper">
+          <img class="img2" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/grass.jpg"/>
+          <div>IMAGE TWO</div>
+        </div>
+        <div class="img3wrapper">
+          <img class="img3" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/glacier_water.jpg"/>
+          <div>IMAGE THREE
+          </div>
+        </div>
+      </div>
+    <br>
+      <div class="button">
+        <button class="button1" type="button"><a target="_blank" href="blog-post.html">Tell Me More!</a></button>
+      </div>
+    <br>
+      <div class="quote">
+        <div class="quote-text">
+        <h3>Use this space for a great quote.</h3>
+        </div>
+      </div>
+
+      <!-- Main Content Page 1-->
+
+      <div class="main-content1-wrapper">
+        <div class="main-content1">
+          <h4 class="main-content1-text1">I DON'T WANT TO SAY <b>IT'S THE ALIENS...</b><br>BUT IT'S THE ALIENS.</h4>
+          <img class="main-content1-img1" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/satellite-aliens.jpg"/>
+          <h3>TOTALLY...</h3>
+          <p class="img1-desc1">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsa eveniet atque placeat neque quasi vel, facilis ullam nemo veritatis, amet, exercitationem officiis consectetur totam. Deleniti porro ipsa optio obcaecati minima placeat saepe officiis accusamus animi amet, voluptates dolorem perspiciatis, culpa earum repellat facere. Iusto, accusantium velit labore nesciunt ea ullam.</p>
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>
+          <br>
+          <p class="underline">                                                            </p>
+          <h4 class="main-content1-text1">AWESOME DEMO PHOTO</h4>
+          <img class="main-content1-img1" src="https://picsum.photos/300/203"/>

```

Comment:
I love the idea of using picsum! Change it up on each refresh!

Reply:
Picsum is really cool! I just changed it up to local images because it was making my eyes hurt every time I refreshed my browser haha. I switched it back to all images.